crates/pevm/tests/mainnet.rs

```diff
@@ -35,14 +34,20 @@ async fn mainnet_blocks_from_rpc() {
     ] {
         let provider = ProviderBuilder::new().on_http(rpc_url.clone());
         let block = provider
-            .get_block(BlockId::number(block_number), BlockTransactionsKind::Full)
+            .get_block(
+                alloy_rpc_types::BlockId::number(block_number),
```

Comment:
This is even longer than `alloy_provider`. For consistency can we `use` it as well 🙏?